pkgs/tools/cd-dvd/ventoy-bin/default.nix

```
@@ -0,0 +1,107 @@
+{
+  lib, stdenv, fetchurl, fetchpatch,
+  autoPatchelfHook, makeWrapper,
+  hexdump, exfat, dosfstools, xz, util-linux, bash, parted
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ventoy-bin";
+  version = "1.0.47";
+  meta = with lib; {
```

```suggestion
  version = "1.0.47";

  meta = with lib; {
```
and please move meta to the end of the file.